Cleanup: Interface - Remove unused functions

These have been around for a while, time to remove them.


```
  575 | static void panel_custom_pin_to_last_set(const bContext *C, const Panel *panel, const bool value)
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
[27/1600] Building CXX object source/blender/editors/interface/CMakeFiles/bf_editor_interface.dir/templates/interface_template_cache_file.cc.o
/mnt/nvme0n1/git/Bforartists/source/blender/editors/interface/templates/interface_template_cache_file.cc: In function ‘void blender::ui::template_cache_file_time_settings(Layout*, blender::PointerRNA*)’:
/mnt/nvme0n1/git/Bforartists/source/blender/editors/interface/templates/interface_template_cache_file.cc:55:17: warning: variable ‘col’ set but not used [-Wunused-but-set-variable]
   55 |   Layout *row, *col; /* BFA , added *col, removed *sub, *subsub */
```

--- a/source/blender/editors/interface/interface_panel.cc
+++ b/source/blender/editors/interface/interface_panel.cc
@@ -571,18 +571,7 @@ static bool panel_custom_pin_to_last_get(const Panel *panel)
   return false;
 }
 
-/* bfa - TODO: Unused - we should use the toggle itself to unpin */
-static void panel_custom_pin_to_last_set(const bContext *C, const Panel *panel, const bool value)
-{
-  if (panel->type->pin_to_last_property[0] != '\0') {
-    PointerRNA *ptr = panel_custom_data_get(panel);
-    if (ptr != nullptr && !RNA_pointer_is_null(ptr)) {
-      PropertyRNA *prop = RNA_struct_find_property(ptr, panel->type->pin_to_last_property);
-      RNA_boolean_set(ptr, panel->type->pin_to_last_property, value);
-      RNA_property_update(const_cast<bContext *>(C), ptr, prop);
-    }
-  }
-}
+/* bfa - REMOVED: panel_custom_pin_to_last_set - we use the toggle itself to unpin */
 
 static bool panel_custom_data_active_get(const Panel *panel)
 {
@@ -2351,8 +2340,7 @@ static void handle_panel_header(const bContext *C,
     const float drag_area_xmax = block->rect.xmax;
     if (IN_RANGE(mx, drag_area_xmin, drag_area_xmax)) {
       if (panel_custom_pin_to_last_get(panel)) {
-        /* bfa - we use the toggle itself to unpin */
-        /* panel_custom_pin_to_last_set(C, panel, false); */
+        /* bfa - REMOVED: panel_custom_pin_to_last_set - we use the toggle itself to unpin */
         return;
       }
       panel_activate_state(C, panel, PANEL_STATE_DRAG);

--- a/source/blender/editors/interface/templates/interface_template_cache_file.cc
+++ b/source/blender/editors/interface/templates/interface_template_cache_file.cc
@@ -52,12 +52,11 @@ void template_cache_file_time_settings(Layout *layout, PointerRNA *fileptr)
   /* Ensure that the context has a CacheFile as this may not be set inside of modifiers panels. */
   layout->context_ptr_set("edit_cachefile", fileptr);
 
-  Layout *row, *col; /* BFA , added *col, removed *sub, *subsub */
+  Layout *row; /* BFA - row */
 
   row = &layout->row(false);
 
   /*------------------- bfa - original props */
-  col = &layout->column(true);
   row = &layout->row(true);
   row->use_property_split_set(false); /* bfa - use_property_split = False */
   row->prop(fileptr, "is_sequence", UI_ITEM_NONE, std::nullopt, ICON_NONE);

